Tools/wasm/testrunner.py

```diff
@@ -0,0 +1,221 @@
+#!/usr/bin/env python3
```

Comment:
Did you mean to include this new file in this PR?

Reply:
Nope